mmv1/products/accesscontextmanager/ServicePerimeterEgressPolicy.yaml

```diff
@@ -0,0 +1,143 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Resource
+name: 'ServicePerimeterEgressPolicy'
+create_url: '{{perimeter}}'
+base_url: ''
+self_link: '{{perimeter}}'
+create_verb: :PATCH
+delete_verb: :PATCH
+update_mask: true
+identity:
+  - egressPolicy
+nested_query: !ruby/object:Api::Resource::NestedQuery
+  modify_by_patch: true
+  is_list_of_ids: true
+  keys:
+    - status
+    - egressPolicies
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/access-context-manager/docs/reference/rest/v1/accessPolicies.servicePerimeters#egresspolicy'
+description: |
+  EgressPolicies match requests based on egressFrom and egressTo stanzas. 
+  For an EgressPolicy to match, both egressFrom and egressTo stanzas must be matched. 
+  If an EgressPolicy matches a request, the request is allowed to span the ServicePerimeter 
+  boundary. For example, an EgressPolicy can be used to allow VMs on networks 
+  within the ServicePerimeter to access a defined set of projects outside the 
+  perimeter in certain contexts (e.g. to read data from a Cloud Storage bucket 
+  or query against a BigQuery dataset).
+autogen_async: true
+exclude_validator: true
+# Skipping the sweeper due to the non-standard base_url and because this is fine-grained under ServicePerimeter/IngressPolicy
+skip_sweeper: true
+id_format: '{{perimeter}}'
+import_format: ['{{perimeter}}']
+custom_code: !ruby/object:Provider::Terraform::CustomCode
```

Comment:
Can we add non-test examples like in https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/accesscontextmanager/ServicePerimeterResource.yaml#L58? So we have examples in our documentation?

Reply:
Sure will do